packages/hdwallet-native/src/native.ts

```diff
@@ -33,11 +35,13 @@ type LoadDevice = Omit<core.LoadDevice, "mnemonic"> & {
 } & (
     | {
         mnemonic: string | Isolation.Core.BIP39.Mnemonic;
-        masterKey?: never;
+        secp256k1MasterKey?: never;
+        ed25519MasterKey?: never;
       }
     | {
         mnemonic?: never;
-        masterKey: Isolation.Core.BIP32.Node;
+        secp256k1MasterKey: Isolation.Core.BIP32.Node;
+        ed25519MasterKey?: Isolation.Core.Ed25519.Node;
```

Comment:
backwards compatible as is, but happy to and feels like we may want to make this required and just have this version include breaking changes. thoughts @gomesalexandre?

currently you can initialize a native wallet without an ed25519 even though there would technically be solana support resulting in the `needsMnemonic` to throw for all solana tx calls requiring a key.

Reply:
I made the call to make it required. We are using mnemonic across the board in web anyways, so it wont directly affect us regardless.